src/compute-client/src/controller.rs

```diff
@@ -59,31 +58,25 @@ pub struct ComputeSinkId {
     pub global_id: GlobalId,
 }
 
-/// Controller state maintained for each compute instance.
+/// A controller for a compute instance.
 #[derive(Debug)]
-pub struct ComputeControllerState<T> {
+pub struct ComputeController<T> {
```

Comment:
Nit: IMO, we should take advantage of this opportunity to change the name, e.g. to "ComputeInstanceController". It's too confusing to have both "controller" (which is global) and "compute controller", which is per-instance.

What do you think?

Reply:
I personally don't like excessively long names because I find them harder too read (we'd end up with `ActiveComputeInstanceController` as well). But I agree with the current situation being confusing as well.

Which makes me wonder why we have a separate controller for every compute instance in the first place? That's inconsistent with the storage controller, which only has a single instance, which certainly adds to the confusion.